src/index.js

```diff
@@ -17,9 +17,14 @@ angular
     translate,
   ])
   .service('ChatbotService', service)
-  .directive('ovhChatbot', directive)
+  .component('ovhChatbot', component)
   .component('ovhChatbotIcons', { template: iconsTemplate })
   .run(/* @ngTranslationsInject:xml ./translations */)
+  .constant('CHATBOT_MESSAGE_QUALITY', {
+    normal: 'normal',
```

Comment:
You could have a `chatbot.constants` file and just import constant when you need